packages/common/src/core/types/IHubProjectView.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Hub Project view: defines the display properties of a project
+ */
+
```

Comment:
I think this extra line will prevent the jsdoc from associating the comment with the interface